Live-Snap/Live-Snap/Networking/SnapcodeEndpointRequest.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  SnapcodeEndpointRequest.swift
+//  Live-Snap
+//
+//  Created by Oleg Abalonski on 1/6/18.
+//  Copyright © 2018 Oleg Abalonski. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+class SnapcodeEndpointRequest {
+    
+    var snapchatUsername: String
+    
+    init(snapchatUsername: String) {
+        self.snapchatUsername = snapchatUsername
+    }
+    
+    func start(completion: @escaping ((UIImage?) -> ())) {
+        
+        let url = "https://feelinsonice-hrd.appspot.com/web/deeplink/snapcode?username=\(snapchatUsername)&type=PNG&size=800"
+    
+        Alamofire.request(url).responseData { (response: DataResponse<Data>) in
+
+            response.result.ifSuccess {
+                if let imageData = response.result.value {
```

Comment:
you need an else case, otherwise this will never call completion if for some reason response.result.value is nil